app/base/views.js

```diff
@@ -27,10 +27,14 @@ exports.error_test = function (req, res, next) {
 
 exports.auth_callback = [
   passport.authenticate('auth0-oidc'),
-  function (req, res) {
+  function (req, res, next) {
     raven.setContext({user: req.user});
     api.set_token(req.user.id_token);
-    res.redirect(req.session.returnTo || '/');
+    api.users.get(req.user.sub)
```

Comment:
So this will depend on James' work here: https://github.com/ministryofjustice/analytics-platform-control-panel/pull/57

Reply:
Yes. It doesn't depend on him implementing it, because we're not doing anything with the response, but he is implementing the user creation in the API.